### ISSUE TITLE
Craftable chests and/or team chests

### ISSUE THREAD

**Issue by fanofMinecraftUa** [MT-CTF/capturetheflag#1124]
When game lasts long, the items can't fit in the main chest, so we need craftable ones. 

- Chests should be breakable with axe, crafted like in base game, can be opened by enemies and placed on their territory.

- Team chests should be crafted from normal chest with 4 iron ingots on sides or on all slots (idk, needs to be balanced). On top of normal _team_ chest mechanics craftable team chests need to be broken with pickaxe, placing mechanics are like in spikes(be white in invertory, and can't be placed on other team side)

Also not related, but how to create a map, I have cool birch forest idea from my dream

**Comment by LuismiInTheZone** [comment]
> When game lasts long, the items can't fit in the main chest, so we need craftable ones.
> 
> * Chests should be breakable with axe, crafted like in base game, can be opened by enemies and placed on their territory.
> * Team chests should be crafted from normal chest with 4 iron ingots on sides or on all slots (idk, needs to be balanced). On top of normal _team_ chest mechanics craftable team chests need to be broken with pickaxe, placing mechanics are like in spikes(be white in invertory, and can't be placed on other team side)
> 
> Also not related, but how to create a map, I have cool birch forest idea from my dream

It seems interesting in some way, but looking to other ways like that people can steal and hide out shotguns and snipers... If you want to learn how to create a map read the readme of the maps github, if not I recomend you joining the ctf discord if you aren't in it. (maps github: https://github.com/MT-CTF/maps)

**Comment by farooqkz** [comment]
I disagree with these chests placed in enemy territory. But enemies being able to steal from them or even put items in them seems reasonable. Like they are just normal chests, except that you can flood enemy territory with them.
